test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/ResponseTests.cs

```diff
@@ -909,7 +919,7 @@ public Task ResponseStatusCodeSetBeforeHttpContextDisposedRequestMalformedReadIg
                         "POST / HTTP/1.1",
                         "Transfer-Encoding: chunked",
                         "",
-                        "wrong");
+                        "gg");
```

Comment:
Unrelated but piggybacking this here. Test can fail if `Send` is still sending bytes but the server has already closed the connection because of the bad chunk length.